content/docs/start/metrics-parameters-plots.md

```diff
@@ -94,9 +97,9 @@ Similarly, it writes arrays for the
 [roc_curve](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html)
 into `roc.json` for an additional plot.
 
-> DVC doesn't force you to use any specific file names, or even format or
-> structure of a metrics or plots file - it's pretty much user and case defined.
-> Please refer to `dvc metrics` and `dvc plots` for more details.
+> DVC doesn't force you to use any specific file names, nor does it enforce a
+> format or structure of a metrics or plots file. It's completely user- and
```

Comment:
That hyphen looks strange?
```suggestion
> format or structure of a metrics or plots file. It's completely user-and
```

Reply:
no, just avoiding suffix repetition (as with spoken English) to keep the tone light. Otherwise would have to say "user-defined and case-defined."

Reply:
> There's a space between "user-" and "and"

That's the strange part. Looks a bit like a typo. Prob best to use the long form you mentioned then.